TestsCalculator/CalculatorPageTests.cs

```diff
@@ -90,13 +320,46 @@ public void PositiveSelectDate()
             percentField.SendKeys("10");
             termField.SendKeys("20");
             dayDropdown.SendKeys("7");
-            monthDropdown.SendKeys("Jun");
+            monthDropdown.SendKeys("Jun"); // why does not fail?
```

Comment:
It's better to work with dropdowns via SelectElement:
```suggestion
            new SelectElement(monthDropdown).SelectByText("June");
```